components/tikv_util/src/config.rs

```diff
@@ -689,6 +689,88 @@ pub fn check_max_open_fds(expect: u64) -> Result<(), ConfigError> {
     }
 }
 
+#[macro_export]
+macro_rules! rocksdb_enum_mod {
+    ($name:ident $enum:ident { $($variant:ident = $value:expr, )* }) => {
+        pub mod $name {
+            use std::fmt;
+
+            use serde::{Serializer, Deserializer};
+            use serde::de::{self, Unexpected, Visitor};
+            use crate::$enum;
```

Comment:
How about this, more flexible.
```suggestion
            use super::$enum;
```

Reply:
Got it